test/com/facebook/buck/python/PythonBinaryIntegrationTest.java

```diff
@@ -126,6 +141,10 @@ public void commandLineArgs() throws IOException {
     assertThat(
         result.getStdout(),
         containsString("HELLO WORLD"));
+
+    if (pexDirectory) {
```

Comment:
It seems a bit weird to test for this in the `commandLineArgs` test.  Maybe move this to a separate `testOutput` test where we check for a directory for the `--directory` case and files for the others?